src/App.css

```diff
@@ -40,3 +40,43 @@
   padding: 0.5rem 1rem;
   color: rgba(0,0,0,.5);
 }
+
+
+/* Moet worden toegevoegd om material icons werkend te maken. */
```

Comment:
maybe change this to English?

Reply:
wow that was my original comment do not steal